server/routes/profile.js

```diff
@@ -15,4 +15,14 @@ router.get('/user/:id', (req, res) => {
     .catch((err) => console.error(err));
 });
 
+
+router.get('/user/:id/badges', (req, res) => {
+  const { id } = req.params;
+  getUserBadges(id)
+    .then((badge) => {
```

Comment:
a bit nitpicky here, but can you make the badge response plural? since we are getting all the badges